### PR TITLE
Draft: Update

### PR DESCRIPTION
**Pull request by forsvunnet**

### CODE REVIEW

File: classes/common/class-checkout-modifications.php [hunk 1]
```diff
@@ -217,4 +236,46 @@ public static function attach_order_note( $order_id ) {
...
+		}
+	}
+
+	/**
+	 * Save user selected bag on door value to order meta
+	 */
+	public static function bag_on_door_order_meta( $order_id ) {
+		$name = "bag_on_door_consent";
+
+		if ( ! empty( $_POST[$name] ) ) {
```
Comment (forsvunnet) [inline classes/common/class-checkout-modifications.php]: Please don't access request variables directly. Use [filter_input](https://www.php.net/manual/en/function.filter-input.php) instead.

File: classes/common/class-checkout-modifications.php [hunk 2]
```diff
@@ -217,4 +236,46 @@ public static function attach_order_note( $order_id ) {
...
+			), WC()->checkout->get_value($name));
+			echo '</div>';
+		}
+	}
+
+	/**
+	 * Save user selected bag on door value to order meta
+	 */
+	public static function bag_on_door_order_meta( $order_id ) {
+		$name = "bag_on_door_consent";
```
Comment (forsvunnet) [inline classes/common/class-checkout-modifications.php]: Ambiguous naming. Also, try not to assign machine keys or slugs to variables. Instead use the full string where it's needed.

File: classes/common/class-checkout-modifications.php [hunk 3]
```diff
@@ -217,4 +236,46 @@ public static function attach_order_note( $order_id ) {
...
+	}
+
+	/**
+	 * Display order related bag on door value in WC order admin page
+	 */
+	public static function bag_on_door_admin_value( $order ) {
+		$consent = get_post_meta( $order->get_id(), "bag_on_door_consent", true );
+
+		if ( $consent == 1 ) {
+			echo '<p><strong>Bag on door: </strong> Enabled</p>';
```
Comment (forsvunnet) [inline classes/common/class-checkout-modifications.php]: Missing translation

Comment (forsvunnet) [reply classes/common/class-checkout-modifications.php]: Also unescaped html. You should use esc_html_e() for this. Pro tip: Use [WordPress-Coding-Standards](https://github.com/WordPress/WordPress-Coding-Standards)

File: classes/common/class-checkout-modifications.php (hunk 1 above)
Comment (burdulixda) [reply classes/common/class-checkout-modifications.php]: Gotcha.

File: classes/common/class-checkout-modifications.php (hunk 3 above)
Comment (burdulixda) [reply classes/common/class-checkout-modifications.php]: Fixed both issues but couldn't get WPCS working on VSC. Maybe you can provide your rulesets?

File: classes/common/class-checkout-modifications.php [hunk 4]
```diff
@@ -217,4 +236,46 @@ public static function attach_order_note( $order_id ) {
...
+				'required'	=> true
+			), WC()->checkout->get_value($name));
+			echo '</div>';
+		}
+	}
+
+	/**
+	 * Save user selected bag on door value to order meta
+	 */
+public static function bag_on_door_order_meta( $order_id ) {
```
Comment (forsvunnet) [inline classes/common/class-checkout-modifications.php]: indentation missing

**Pull request merged**